tmx_object.go

```diff
@@ -153,3 +155,18 @@ func (m *Points) UnmarshalXML(d *xml.Decoder, start xml.StartElement) error {
 	*m = (Points)(points)
 	return nil
 }
+
+// Text contains a text and attributes such as bold, color, etc.
+type Text struct {
+	Text       string `xml:",chardata"`
+	Fontfamily string `xml:"fontfamily,attr"`
+	Pixelsize  int    `xml:"pixelsize,attr"`
```

Comment:
Field name should be `Size`

Reply:
Changed it to Size, or did you mean PixelSize?